test/services/leaderboard-stat-getter.test.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable import/first */
+jest.mock('forest-express');
+jest.mock('../../src/utils/orm');
+import Sequelize from 'sequelize';
+import ForestExpress from 'forest-express';
+import Orm from '../../src/utils/orm';
+
+const sequelize = new Sequelize({ dialect: 'postgres' });
+
+const address = sequelize.define('address', {
+  id: {
+    type: Sequelize.DataTypes.INTEGER,
+    primaryKey: true,
+  },
+  user_id: Sequelize.DataTypes.STRING,
+});
+
+const user = sequelize.define('user', {
+  id: {
+    type: Sequelize.DataTypes.INTEGER,
+    primaryKey: true,
+  },
+  username: Sequelize.DataTypes.STRING,
+});
+
+address.belongsTo(user, {
+  foreignKey: {
+    name: 'userIdKey',
+    field: 'user_id',
+  },
+  as: 'user',
+});
+
+user.hasMany(address, {
+  foreignKey: {
+    name: 'userIdKey',
+    field: 'user_id',
+  },
+  as: 'addresses',
+});
+
+address.primaryKeys = Object.keys(address.primaryKeys);
+
+ForestExpress.Schemas = { schemas: { user, address } };
+
+jest.spyOn(Orm, 'getColumnName').mockReturnValue('id');
+
+import LeaderBoardStatGetter from '../../src/services/leaderboard-stat-getter';
+
+describe('services > leaderboard-stat-getter', () => {
+  it('should build the right query', async () => {
```

Comment:
[TEST SUITES] I think we are to general with our test labels. Of course this test is helpful, however you are adding some sort of non regression test. Having the issue fixed clearly described somehow in the test label would be really helpful.

Reply:
```suggestion
  it('identifier should be double quoted', async () => {
```